keyboards/fleuron/readme.md

```diff
@@ -0,0 +1,36 @@
+fleuron keyboard firmware
```

Comment:
You're using a very out of date version of the read me here, actually.  

Could you use this instead: https://github.com/qmk/qmk_firmware/blob/master/quantum/template/base/readme.md

Reply:
Done :)